Resources/Prototypes/_NF/Entities/Markers/shuttle.yml

```diff
@@ -0,0 +1,12 @@
+- type: entity
+  parent: IDCardStandard
+  id: ShuttleDeedIDCard
+  name: shuttle deed
+  components:
+  - type: Sprite
+    layers:
+    - state: default
+    - state: idpassenger
+  - type: PresetIdCard
+    job: Captain
+  - type: StationDeedSpawner
```

Comment:
needs to have a suffix of [DO NOT MAP EVER] or similar way to prevent mappers from using this.

Reply:
Added the classic DO NOT MAP